### PR TITLE
:book: update doc samples

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Description
update doc samples

## Motivation

Try to keep the samples sync with the latest changes

### CODE REVIEW

File: docs/book/src/cronjob-tutorial/testdata/project/main.go [hunk 1]
```diff
@@ -107,29 +89,16 @@ func main() {
 		os.Exit(1)
 	}
 
-	// +kubebuilder:docs-gen:collapse=old stuff
```
Comment (camilamacedo86) [inline docs/book/src/cronjob-tutorial/testdata/project/main.go:110]: /hold

Required came back this

File: docs/book/src/multiversion-tutorial/testdata/project/Makefile [hunk 2]
```diff
@@ -40,11 +40,7 @@ help: ## Display this help.
 
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-    # Note that the option maxDescLen=0 was added in the default scaffold in order to sort out the issue
-    # Too long: must have at most 262144 bytes. By using kubectl apply to create / update resources an annotation 
-    # is created by K8s API to store the latest version of the resource ( kubectl.kubernetes.io/last-applied-configuration). 
-    # However, it has a size limit and if the CRD is too big with so many long descriptions as this one it will cause the failure. 
-	$(CONTROLLER_GEN) rbac:roleName=manager-role crd:maxDescLen=0 webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
```
Comment (camilamacedo86) [inline docs/book/src/multiversion-tutorial/testdata/project/Makefile:43]: Came back this

**Comment by camilamacedo86** [comment]
/hold cancel

It is ok to get merged now.

**Review by everettraven — APPROVED** [review]
These all look good and make sense to me

/lgtm

**Pull request merged**